tsl/test/sql/CMakeLists.txt

```diff
@@ -69,6 +69,7 @@ if(CMAKE_BUILD_TYPE MATCHES Debug)
     dist_api_calls.sql
     dist_commands.sql
     dist_compression.sql
+    dist_copy_big.sql
```

Comment:
Suggest to add an option to allow "big" tests to be excluded from the test suite. Should probably default to "false" so that we do not run these by default when developing, but enable them in the CI.

Reply:
+1

Reply:
I'll try to do that in a separate PR. Let's use "long" suffix in the name for them.